src/commands.cpp

```diff
@@ -3579,6 +3584,13 @@ void CommandGetUserData::procresult()
             }
             break;
 
+        case MAKENAMEID4('s', 'm', 's', 'v'):
+            if (!client->json.storeobject(&smsv))
+            {
+                return client->app->userdata_result(NULL, NULL, NULL, API_EINTERNAL);
```

Comment:
Fixed

Reply:
Ok thanks, but I feel it may be better for the original author to make changes. That way it'd be easier to follow and understand as to what's going on.